invesalius/gui/task_mepmapping.py

```diff
@@ -0,0 +1,346 @@
+# --------------------------------------------------------------------------
```

Comment:
This file can be removed, right?

Reply:
Not sure as I was planning to add the import/export ui buttons here..

Reply:
The import/export markers can be done using the save/load markers from the navigation tab

Reply:
But the exporting of colored stl is quite interesting

Reply:
Yes I might do it in a future pull request so that is why I feel like I am going to need the file.. Or do we just delete it and add it later when we do the feature

Reply:
I think you can keep this file here, but please, add a TODO note